src/vaults/vaults.utils.ts

```diff
@@ -335,6 +336,8 @@ export async function loadVaultEventPerformances(
     const { sett } = await getVault(chain.graphUrl, vaultDefinition.vaultToken, end.block);
     if (sett) {
       weightedBalance += duration * formatBalance(sett.balance, depositToken.decimals);
+    } else {
+      weightedBalance += duration * vault.balance;
```

Comment:
have a pessimistic fallback option in the case the graph is out of sync or failing